src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -475,7 +468,6 @@ const formConfig = {
               ({ formData }) => `${sponsorWording(formData)} mailing address`,
               'We’ll send any important information about this application to this address.',
             ),
-            ...homelessInfo.uiSchema,
```

Comment:
`homelessInfo` is in the same boat as `noPhoneInfo` - we thought we'd bring it back, but we aren't.